src/bin/hexyl.rs

```diff
@@ -93,15 +100,24 @@ fn run() -> Result<(), Box<dyn std::error::Error>> {
         None => Input::Stdin(stdin.lock()),
     };
 
-    let skip_arg = matches.value_of("skip").and_then(parse_hex_or_int);
+    let block_size = matches
+        .value_of("block_size")
+        .and_then(|bs| bs.parse::<u64>().ok())
+        .unwrap_or(512);
+
+    let skip_arg = matches
+        .value_of("skip")
+        .and_then(|s| parse_byte_count(s, block_size));
 
     if let Some(skip) = skip_arg {
         reader.seek(SeekFrom::Start(skip))?;
     }
 
+    #[allow(clippy::or_fun_call)]
```

Comment:
Why disable the clippy lint rather than just fixing it with `.or_else(|| matches.value_of("bytes"))`? (same below)

Reply:
I simply disabled the lint as a conservative effort to preserve the eagerness of the expression being evaluated. I'm not positive a behavior change would result, but I just avoided the question altogether.

Reply:
Using `.or_else` is fine here.

Reply:
Fixed.